### ISSUE TITLE
`unsigned` modifier on `__INT32_TYPE__ ` unsupported

### ISSUE THREAD

**Issue by andrewvaughanj** [ltcmelo/psychec#35]
#### GCC version:
```bash
gcc (SUSE Linux) 10.2.1 20201202 [revision e563687cf9d3d1278f45aaebd03e0f66531076c9]
```

#### clang version:
```bash
clang version 11.0.1
```

#### Output of `gcc -Werror -Wall -Wextra -Wpedantic -pedantic -std=c11 -c $FNAME`
```bash
(return code 0)
```

#### Output of `clang -Werror -Wall -Wextra -Wpedantic -pedantic -std=c11 -c $FNAME`
```bash
(return code 0)
```

#### Output of `cnip $FNAME`
```bash
llvm.c:0:24 error: expected `,' or `;' or `=' after declarator, got `uint32_t'
unsigned __INT32_TYPE__ uint32_t;
                        ^

(return code 0)
```

#### Content of file
```bash
unsigned __INT32_TYPE__ uint32_t;
```

#### `psychec` version
```bash
commit d10dc1ddfb7ea845f59274a4d6ca5d021ab31423
Author: Leandro T. C. Melo <LTCMELO@GMAIL.COM>
Date:   Tue Mar 9 21:18:30 2021 -0300

    fix postfixing on compound literal
```

**Comment by andrewvaughanj** [comment]
I think this "goes away" if you use the preprocessor (see #40):

```bash
avj@platypus ~/clones/psychec/preprocessor/build$ cat /tmp/moo/b.c
unsigned __INT32_TYPE__ uint32_t;
avj@platypus ~/clones/psychec/preprocessor/build$ ./cnip /tmp/moo/b.c; echo $?
b.c:0:24 error: expected `,' or `;' or `=' after declarator, got `uint32_t'
unsigned __INT32_TYPE__ uint32_t;
                        ^

0
avj@platypus ~/clones/psychec/preprocessor/build$ ./cnip --cc-pp /tmp/moo/b.c; echo $?
0
```

**Comment by ltcmelo** [comment]
Fixed with #62 .
